src/controllers/llm.ts

```
@@ -143,29 +138,19 @@ export class LLMController {
         }
 
         // Remove headers that might cause issues
-        if (axiosConfig.headers != null) {
-          delete axiosConfig.headers['host']
-          delete axiosConfig.headers['content-length']
-        }
+        delete axiosConfig.headers['host']
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)